scripts/install

```diff
@@ -1,202 +1,234 @@
+	cp ../conf/backend/config-caldav.php $final_path/backend/caldav/config.php
+	cp ../conf/backend/config-carddav.php $final_path/backend/carddav/config.php
+	cp ../conf/backend/config-imap.php $final_path/backend/imap/config.php
+	cp ../conf/backend/config-combined.php $final_path/backend/combined/config.php
+else
+	# Configuration of backend
+	ynh_replace_string __BACKEND__ BackendIMAP ../conf/config.php
+	ynh_replace_string __BACKEND__ BackendIMAP ../conf/backend/config-autodiscover.php
 
+	# Configuration imap
+	ynh_replace_string __IMAP_SERVER__ "${domain}" ../conf/backend/config-imap.php
+	ynh_replace_string __FLAGTOCHANGE__ false ../conf/backend/config-imap.php
+
+	# Copy config
+	cp ../conf/backend/config-imap.php $final_path/backend/imap/config.php	
 fi
 
-# Set permissions to z-push directory
-sudo chown -R www-data: $final_path
+# Copy config
+cp ../conf/config.php $final_path/config.php
+cp ../conf/backend/config-autodiscover.php $final_path/autodiscover/config.php
+cp ../conf/backend/config-searchldap.php $final_path/backend/searchldap/config.php
 
-# Create log directory
-sudo mkdir -p $final_logpath
-sudo chmod 750 $final_logpath
-sudo chown www-data:www-data $final_logpath
+#Copy XMLElement.php
+ln -s /usr/share/awl/inc/XML* /var/www/$app/include/
 
-# Activate logrotate
-sudo cp ../conf/logrotate /etc/logrotate.d/$app
-sudo chmod 644 /etc/logrotate.d/$app
-sudo chown root:root /etc/logrotate.d/$app
+#=================================================
+# GENERIC FINALIZATION
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
 
-#Copy XMLElement.php
-sudo ln -s /usr/share/awl/inc/XML* /var/www/$app/include/
-
-# Modify Nginx configuration file and copy it to Nginx conf directory
-sudo sed -i "s@ALIASTOCHANGE@$final_path/@g" ../conf/nginx.conf
-sudo sed -i "s@NAMETOCHANGE@$app@g" ../conf/nginx.conf
-sudo cp ../conf/nginx.conf $final_nginxconf
-
-# Dedicated php-fpm processes
-sudo sed -i "s@NAMETOCHANGE@$app@g" ../conf/php-fpm.conf
-sudo cp ../conf/php-fpm.conf $final_phpconf
-sudo chown root: $final_phpconf
-sudo chmod 644 $final_phpconf
-
-# Enable api for client and make app public
-ynh_app_setting_set $app skipped_uris "/"
-
-# Reload Nginx and regenerate SSOwat conf
-sudo service php5-fpm reload
-sudo service nginx reload
-sudo yunohost app ssowatconf
+# Set permissions to app files
+chown -R www-data: $final_path
+
+#=================================================
+# SETUP LOGROTATE
+#=================================================
+ynh_print_info "Configuring log rotation..."
+
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
+
+#=================================================
+# SETUP SSOWAT
+#=================================================
+ynh_print_info "Configuring SSOwat..."
+
+# Make app public
+ynh_app_setting_set $app unprotected_uris "/"
+ynh_app_setting_set $app skipped_regex "^[^/]*/[Aa]uto[Dd]iscover.*"
```

Comment:
Using `skipped_regex` will match any path on the server, not only those for this app.
You should put the domain and path at the beginning of the path to restrict the regex to this app.

Reply:
Change made